meson.build

```diff
@@ -1070,7 +1070,10 @@ x11 = {
              dependency('xscrnsaver', version: '>= 1.0.0', required: x11_opt),
              dependency('xext', version: '>= 1.0.0', required: x11_opt),
              dependency('xpresent', version: '>= 1.0.0', required: x11_opt),
-             dependency('xrandr', version: '>= 1.4.0', required: x11_opt)],
+             dependency('xrandr', version: '>= 1.4.0', required: x11_opt),
+             dependency('x11-xcb', version: '>= 1.8.9', required: x11_opt),
+             dependency('xcb-shm', version: '>= 1.16', required: x11_opt),
+             dependency('xcb', version: '>= 1.16', required: x11_opt),]
```

Comment:
This isn't available on Debian Stable, and considering the whole purpose of this VO is for compatibility with old systems, I'm not sure we want to require this to build X11 at all. Can this be an optional dep?

Reply:
XCB only seems to be used for a single call that could probably be done without XCB.